ivy_tests/test_ivy/test_frontends/test_tensorflow/test_tf_functions.py

```diff
@@ -189,3 +189,36 @@ def test_tensorflow_full(
         value=fill_value,
         rtol=1e-05,
     )
+
+
+# subtract
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=tuple(
+            set(ivy_np.valid_float_dtypes).intersection(set(ivy_tf.valid_float_dtypes))
```

Comment:
We should use `ivy_np.valid_numeric_dtypes `and `ivy_tf.valid_numeric_dtypes`. As subtract will accept int dtypes

Reply:
Hi @jkeane508
Fixed!
Please review

Reply:
Hey, you've actually changed both to `ivy_np.valid_numeric_dtypes` when we should intersect both `ivy_np.valid_numeric_dtypes` and `ivy_tf.valid_numeric_dtypes`

Reply:
Oops! my bad!